packages/libs/error-stack/src/hook.rs

```diff
@@ -87,6 +87,82 @@ impl Report<()> {
     /// <pre>
     #[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/tests/snapshots/doc/hook__debug_hook.snap"))]
     /// </pre>
+    ///
+    /// This example showcases the ability of hooks being invoked for values provided via the
+    /// [Provider API](crate::Context), this feature is currently only available on nightly.
```

Comment:
The link to `Provider API` is pointing to `crate::Context` which is confusing IMO. 
```suggestion
    /// This example showcases the ability of hooks to be invoked for values provided via the
    /// Provider API using [`Error::provide`].
```